src/prefect/client.py

```diff
@@ -1270,10 +1283,14 @@ async def create_deployment(
         schedule: schemas.schedules.SCHEDULE_TYPES = None,
         parameters: Dict[str, Any] = None,
         description: str = None,
+        work_queue_name: str = None,
```

Comment:
```suggestion
        work_queue_name: str ="global",
```
should this be "global" by default?